processor/src/main/java/org/mapstruct/ap/internal/model/source/MappingOptions.java

```diff
@@ -74,7 +80,14 @@ public static MappingOptions empty() {
      * @return MappingOptions with only regular mappings
      */
     public static MappingOptions forMappingsOnly( Map<String, List<Mapping>> mappings ) {
```

Comment:
This is connected with the `isRestrictToDefinedMappings()` discussion. In a symmetric case we want to not explicitly ignore unmapped target properties. However, in assymetric we want to have it explicitly.